src/Composer/ReleaseVersion.php

```diff
@@ -34,6 +34,9 @@ public static function execute(Event $event) {
     foreach ($finder as $info_file) {
       $info = Yaml::parse($info_file->getContents());
 
+      if (@$info['name'] == 'Lightning' && $info['type'] == 'profile') {
```

Comment:
Why is $info['name'] error-suppressed?

Reply:
Not all info files have the `name` and or `type` key. And rather than do two `isset`s, I figured it was acceptable to just suppress in this case.

Reply:
...turns out, this script is picking up the Console translations - those are the "things" without those keys. Will try to fix that instead.